install.sh

```
@@ -0,0 +1,85 @@
+mkdir -p /var/lib/nerd
+mkdir -p /var/log
+
+# Build the project
+echo "Building NERD..."
+if [ ! -d "build" ]; then
+    mkdir build
+fi
+
+cd build
+cmake ..
+make -j$(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)